classes/WP_CLI/ActionScheduler_WPCLI_Scheduler_command.php

```diff
@@ -5,6 +5,37 @@
  */
 class ActionScheduler_WPCLI_Scheduler_command extends WP_CLI_Command {
 
+	/**
+	 * Force tables schema creation for Action Scheduler
+	 *
+	 * ## OPTIONS
+	 *
+	 * @param array $args Positional arguments.
+	 * @param array $assoc_args Keyed arguments.
+	 * @throws \WP_CLI\ExitException When an error occurs.
+	 *
+	 * @subcommand fix-schema
+	 */
+	public function fix_schema( $args, $assoc_args ) {
+		$classes                 = get_declared_classes();
+		$action_scheduler_schema = array();
+
+		foreach ( $classes as $classname ) {
+			if ( is_subclass_of( $classname, 'ActionScheduler_Abstract_Schema' ) ) {
+				$obj = new $classname();
+				$obj->register_tables( true );
+
+				WP_CLI::success(
+					sprintf(
+						/* translators: %d refers to the total number of taskes completed */
```

Comment:
Fix the translator note here? (In the string on the next line we're using `%s` rather than `%d`, and it represents the schema name rather than total tasks completed.)